test_requirements.txt

```diff
@@ -1,3 +1,4 @@
 -r requirements.txt
+cloudpickle>=0.3.1
```

Comment:
Definitely. Will wait for next release to set the appropriate version (they might bump it to 0.4.0).

Maybe they do a release in the future `0.3.2` with just a couple of fixes to the old version but without breaking the `transient` support, so I think we should better wait to know the actual version number... :wink: